source/renderer/app/components/widgets/forms/PinCode.js

```diff
@@ -67,6 +71,28 @@ export default class PinCode extends Component<Props> {
     }
   }
 
+  onKeyPress = (evt: SyntheticKeyboardEvent<EventTarget>, inputKey: string) => {
+    const { key, target } = evt;
+    const control: { blur?: Function, focus?: Function } = target;
+    const focusKey = parseInt(inputKey, 10);
+    const nextFieldFocusKey = focusKey + 1;
+    const nextInputField = this.inputsRef[nextFieldFocusKey];
+    const isDotOrCommaKey = key === ',' || key === '.';
```

Comment:
I think this should use the group & decimal separator from the bignumber.js config:
`const { decimalSeparator, groupSeparator } = BigNumber.config().FORMAT` otherwise different settings could lead to the same bug

Reply:
Makes sense, i have added condition to check separators so we should cover this part now.